cumulus-client/src/main/java/mil/army/usace/hec/cumulus/client/controllers/CumulusDownloadListener.java

```diff
@@ -0,0 +1,16 @@
+package mil.army.usace.hec.cumulus.client.controllers;
+
+import java.io.IOException;
+
+public interface CumulusDownloadListener {
```

Comment:
I'd like to go over this interface with you

Reply:
separated into two separate interfaces. one for generation, and one for file download